spec/features/user_can_add_items_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+# As a user, I'd like to enter an Ikea item to create it on the review site.
+# - User enters a URL or Article Number to an Ikea item
+# - Site creates a page with pertinent information from the Ikea listing
+# -- Item title
+# -- Item subtitle
+# -- Item description
+# -- Picture
+# -- Price
+#
+# - If the item is already created, just redirect user to that page
+# - Page must message users appropriately for invalid URLs / article numbes
+
+feature "User creates a new Ikea item" do
+	scenario "User enters a URL to create a new item" do
+		visit new_item_path
+		expect(page).to have_content("Enter your item here:")
+		expect(page).to have_content("Please use the Ikea URL or the item's article number")
+
+		fill_in "URL", with: "http://www.ikea.com/us/en/catalog/products/S99129122/"
+		click_button "Submit"
+
+		expect(page).to have_content "EKTORP"
+		expect(page).to have_content "Footstool, Nordvalla light blue"
+		expect(page).to have_content "149.00"
+		expect(page).to have_css("img[src*='ektorp-footstool-blue__0386202_PE559152_S4.JPG']")
```

Comment:
What if the Ikea website changes?

Don't need to fix right now, but make a note to come back and figure out a way to handle this